.github/workflows/main.yml

```diff
@@ -108,4 +108,5 @@ jobs:
       - name: Test rollup-http-server and echo-dapp client
         run: |
           cd rollup-http/rollup-http-server
+          echo -e  "[toolchain]\nchannel = \"1.77.2\"" > rust-toolchain.toml
```

Comment:
what is going on here?

Reply:
Workaround for fixing rust toolchain on CI, cause the base image was updated.